docs/azure/conf.py

```diff
@@ -79,8 +79,8 @@
         "../instances/get-ubuntu-pro/",
     "azure-how-to/create-pro-fips-golden-image":
         "../instances/create-pro-fips-golden-image/",
-    "azure-how-to/upgrade-from-focal-to-jammy":
-        "../instances/upgrade-from-focal-to-jammy/",
+    "azure-how-to/upgrade-ubuntu-lts-release":
```

Comment:
```suggestion
    "azure-how-to/upgrade-from-focal-to-jammy":
```
Since this statement is used to redirect old links to new ones, we need to keep the old one as is and only change the new one.

Reply:
Oops. I will fix that. Thanks.